pkgs/top-level/all-packages.nix

```diff
@@ -20619,6 +20619,10 @@ with pkgs;
 
   petsc = callPackage ../development/libraries/science/math/petsc { };
 
+  scs = callPackage ../development/libraries/science/math/scs {
+    liblapack = liblapackWithoutAtlas;
+  };
```

Comment:
Presumably this would be fine with atlas, but I couldn't get atlas to build on my system.  It appears to be a source-only package so that it can do optimizations for your particular host CPU, so there are no Hydra binary caches and I figured ofborg probably wouldn't like it either.